sourcecode/apis/contentauthor/resources/assets/react/components/EditorContainer/EditorContainer.js

```diff
@@ -9,7 +9,7 @@ import { makeStyles } from '@material-ui/core/styles';
 
 const useStyle = makeStyles((theme) => ({
     tab: {
-        backgroundColor: theme.palette.background.paper,
+        backgroundColor: theme.palette.background.paper, textTransform: 'none',
```

Comment:
This is a style change we want everywhere the Tab component is used, this is _currently_ the only one, but that could change. So to avoid overriding this everywhere Tab is, or will be, used, this should be set in the theme.